src/Classes/MyRadio/MyRadioFormField.php

```diff
@@ -784,7 +784,7 @@ private function convertTime($timeString)
          * YES, PHP IS RETARDED.
          */
         $timeString = str_replace('/', '-', $timeString);
-        $time = (int) strtotime($timeString) - date('Z');
+        $time = (int) strtotime($timeString);
```

Comment:
yes and no, requested time is an offset from 1970-01-01 00:00:00 as it is only the hours and mins that are relevant - the date is calculated by whatever uses this, and it is added as an offset to the datetime